# location of obstacles and road in angled segments
float64 rostime
float32[] angle
float32[] obstacle_depth
float32[] obstacle_height_raw
float32[] obstacle_height_relative
float32[] road_start
float32[] road_end
float32[] road_height_raw
float32[] road_height_relative
